### DISCUSSION TITLE
Debt Calculation is wrong by $575?

### DISCUSSION THREAD

**Discussion by Feanan** [JermaSites/JermaDebt#32] [General]
Long story short I wanted to make a graph in Excel of Jerma's debt over time so chucked the numbers given into excel and got Jerma's debt as $3,999,270,714 ! $575 less than what the counter says 

[my workings](https://i.imgur.com/vByB5gx.png)

So firstly the deal or no deal debt double figure looks wrong as

A) the GAS Saw script bet happened an hour BEFORE the debt was doubled

B) Doesn't seem to include the $100 boring man apology removal

So the debt doubling figure should be $2,649,727,215 that is added instead of $2,399,727,315. For some reason only B) affects the figure and not A) so this accounts for $200 and I assume the other $375 has something to do with the '75 community subs' but I'm not sure how 

Thanks for the work which goes into the site

**Comment by sticks-stuff** [comment]
You're completely correct, this is actually something we realized yesterday and have been working to fix. We're currently undergoing a complete rewrite of the site to use a database and to calculate the debt on the fly to avoid issues like this. I got the same figure as you did when calculating it myself. This issue will be rectified later today and an apology will be issued to those affected.
![image](https://user-images.githubusercontent.com/4157860/104140748-c7eb4e00-5417-11eb-92f8-c67b95261dcb.png)

**Comment by sticks-stuff** [comment]
Also, if you're willing to help us create new graphs and visualizations for Jerma's complete and utter gambling addiction, we're always looking for more people to join the jerma.io team and we'd be more than happy to add you to the discord.
